### PR TITLE
Show user activities on user page

### PR DESCRIPTION
**Pull request by ngotungson**

### CODE REVIEW

**Comment by phanbt58** [comment]
ok

File: app/views/static_pages/home.html.erb [hunk 1]
```diff
@@ -9,7 +9,8 @@
 </div>
 <div class="row">
   <div class="col-md-5 col-md-offset-7">
-    <center> <h4> Recently actions on page: </h4> </center>
+    <center> <%= t "views.activities.recent_action" %> </center>
     <%= render partial: "activities/activities", locals: {activities: @activities} %>
```
Comment (hieuhtk58) [inline app/views/static_pages/home.html.erb]: qua 80 ki tu

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -18,8 +18,12 @@
       <%= render "follow_form" if logged_in? %>
     </div>
   </div>
-  <div class="col-md-4">
+  <div class="col-md-3">
     <%= render "list_reading_history" %>
   </div>
+  <div class="col-md-5">
+    <center> <h4> <%= t "views.activities.recent_action" %> </h4> </center>
+    <%= render partial: "activities/activities", locals: {activities: @activities} %>
```
Comment (hieuhtk58) [inline app/views/users/show.html.erb]: qua 80 ki tu

**Comment by hieuhtk58** [comment]
OK

**Pull request merged**